nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/action/TrashAction.java

```diff
@@ -110,12 +111,12 @@ protected void removeProxies(CoreSession session, List<String> ids) {
             }
         }
 
-        public void setSystemProperty(CoreSession session, List<String> ids, Boolean value) {
+        public void setSystemProperty(CoreSession session, List<String> ids, boolean value) {
```

Comment:
don't change the `public` method signature this will break the forward compatibility because a customer code will not compile in a case where the passed param is a `null` value. Actually if a client code call this method with a `null` param it will compile and fail with NPE at line `119` just keep this behaviour.